### PR TITLE
[CDAP-16208] Add generic widgets to storybook

### PR DESCRIPTION
**Pull request by yukiej**
JIRA: https://issues.cask.co/browse/CDAP-16208
Build: https://builds.cask.co/browse/CDAP-UDUT495

Adding abstract widgets to storybook. Also updated config file to allow use of Component Story Format and get i18n info. 

e2e tests: ~https://builds.cask.co/browse/IT-UPD2109~ https://builds.cask.co/browse/IT-UPD2124-1

### CODE REVIEW

File: cdap-ui/.storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<script>
```
Comment (ajainarayanan) [inline cdap-ui/.storybook/preview-head.html:17]: This file needs license header.

**Review by ajainarayanan — COMMENTED** [review]
@yukiej Looks like a rat check is failing causing build failure.

**Comment by yukiej** [comment]
On hold to figure out why the license headers are showing up with the library upgrades.

**Review by ajainarayanan — APPROVED** [review]

**Comment by ajainarayanan** [comment]
@yukiej Couple of comments,
1. The basic widgets like select, textboxes seem different (without borders). It should look the same as it is in the plugins or other places (with labels and borders)
2. Can you file a JIRA to track adding components that we use but are not in the storybook?

**Pull request merged**